core/src/components/picker-column/picker-column.scss

```diff
@@ -112,66 +112,10 @@
   justify-content: start;
 }
 
-// TODO FW-5580 remove this
-:host .picker-item {
```

Comment:
All `.picker-item` selectors should go away since we are using `ion-picker-column-option` instead.